### DISCUSSION TITLE
Not able to use Copilot on VS Code

### DISCUSSION THREAD

**Discussion by mehragaurav** [community/community#124523] [Copilot Conversations]
### Select Topic Area

Question

### Body

I am getting this error while trying to use copilot in vs code.
I have organization access.
- Extension activation failed: "Timed out waiting for authentication provider to register"
- github.copilot.signIn: Error: Non-error thrown: Timed out waiting for authentication provider to register

Tried complete uninstall and reinstall. I am using other git hub extensions like gitlens and git actions all are working also I am signed in using the same account on which I have access. Please help.

**Comment by Rayiumir** [comment]
Hi ... @mehragaurav 

It comes back to the internet connections and sometimes it shows these errors.
Try again.

**Reply by mehragaurav** [reply]
Internet is working fine but I am not able to go beyond these issue...

<img width="1169" alt="image" src="https://github.com/community/community/assets/117749169/40c25cc7-3817-4eab-9cb4-e496b0adcbb4">
